smclip/commands.py

```diff
@@ -379,6 +382,21 @@ def invoke_default(self, raw_args):
     def get_subcmd_real_name(self, subcmd_cls):
         return self._subcmd_names.get(subcmd_cls)
 
+    def commands_to_be_used(self, raw_args):
```

Comment:
a little documentation wouldn't hurt, would it?

Reply:
Added in fe81533. thx